src/Microsoft.ML.LightGbm/LightGbmTrainerBase.cs

```diff
@@ -436,7 +445,7 @@ private protected virtual void GetDefaultParameters(IChannel ch, int numRow, boo
 
         private FloatLabelCursor.Factory CreateCursorFactory(RoleMappedData data)
         {
-            var loadFlags = CursOpt.AllLabels | CursOpt.Features;
+            var loadFlags = CursOpt.AllLabels | CursOpt.AllFeatures;
```

Comment:
Interesting side effect of moving to `CursOpt.AllFeatures` is that feature values of +/-Infinity will also be now be allowed to flow to LightGBM.

How we use the `CursOpt.AllowBadLabels` (called `_keepBad` here) to select rows to keep:
https://github.com/dotnet/machinelearning/blob/af4ca7ce217ca7deac938d2ed4ea1b84d0028b31/src/Microsoft.ML.Data/Training/TrainerUtils.cs#L849-L863

This is good since LightGBM handles infinity nicely. Though we may want a unit test taking in a `Single.PositiveInfinity` as a feature value to LightGBM to train and to score, and seeing that it (or our wrapper code) doesn't throw to test this newfound ability.